src/Workspaces/MSBuild/Core/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -22,25 +22,13 @@
     </PackageDescription>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" />
+    <PackageReference Include="Microsoft.Build.Framework" />
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Newtonsoft.Json" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
     <ProjectReference Include="..\..\Core\Portable\Microsoft.CodeAnalysis.Workspaces.csproj" />
-
-    <!-- For the following package we directly include the binary into this NuGet package because we don't want/need a separate NuGet package
-         for it. PrivateAssets="all" is needed to prevent this reference from becoming a package reference in the package, as a workaround for
-         https://github.com/NuGet/Home/issues/3891.
-         -->
-    <ProjectReference Include="..\BuildHost\Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj" PrivateAssets="all" ReferenceOutputAssembly="false">
-      <IncludeOutputInThisPackage>true</IncludeOutputInThisPackage>
-    </ProjectReference>
-    <ProjectReference Include="..\..\..\Tools\ExternalAccess\RazorCompiler\Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.csproj" PrivateAssets="all">
-      <IncludeOutputInThisPackage>true</IncludeOutputInThisPackage>
-    </ProjectReference>
```

Comment:
Is it safe to remove this ProjectReference now? I'd imagine we need this for compatibility for a very, very long time. In my draft PR I was only deleting the reference to the BuildHost -- this reference didn't exist at the time.

Reply:
Will restore.